### PR TITLE
Update Traditional_Chinese.properties

### PR DESCRIPTION
**Pull request by Jia35**

### CODE REVIEW

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 1]
```diff
@@ -1324,7 +1324,7 @@ Unhappiness modifier = 不滿修正
 Bonus vs. Barbarians = 對戰蠻族時攻擊加成
 Barbarian spawning delay = 蠻族生成延遲
  # Requires translation!
-Bonus starting units = 
+Bonus starting units = 蠻族起始單位
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:1327]: image
Bonus doesn't have to be the babarions.
```suggestion
Bonus starting units = 額外起始單位
```

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 2]
```diff
@@ -4927,11 +4927,11 @@ Modern Armor = 現代坦克
 Jet Fighter = 噴氣式戰鬥機
 
  # Requires translation!
-Giant Death Robot = 
+Giant Death Robot = 巨型死亡機器人
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:4930]: Already forgotton about how Traditional Chinese translator in civ5 translated it, but I know the Simplified_Chinese had a much better translation.
```suggestion
Giant Death Robot = 末日機甲
```

**Review by heipizhu4 — CHANGES_REQUESTED** [review]
Enjoys most of the translation. Just a few suggestions. (I may be slow to reply as I'm going to sleep)

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 3]
```diff
@@ -1521,61 +1521,61 @@ Done! = 完成！
...
+Downloadable mods = 可下載的模組
  # Requires translation!
-Mod info and options = 
+Mod info and options = 模組資訊與選項
  # Requires translation!
-Next page = 
+Next page = 下一步
  # Requires translation!
-Open Github page = 
+Open Github page = 開啟 Github 頁面
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:1536]:
```suggestion
Open Github page = 前往 Github 頁面
```
or：
```suggestion
Open Github page = 打開 Github 頁面
```

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 4]
```diff
@@ -1804,15 +1804,15 @@ Granary = 糧倉
...
  # Requires translation!
-[relativeAmount]% [stat] [cityFilter] = 
+[relativeAmount]% [stat] [cityFilter] = [cityFilter][relativeAmount]%[stat]
 [relativeAmount]% Production when constructing [baseUnitFilter] units [cityFilter] = [cityFilter]建造[baseUnitFilter]單位時[relativeAmount]%產能
 
 The Great Lighthouse = 大燈塔
 'They that go down to the sea in ships, that do business in great waters; these see the works of the Lord, and his wonders in the deep.' - The Bible, Psalms 107:23-24 = “在海上坐船，在大水中經理事務的；他們看見耶和華的作為，並他在深水中的奇事。”——聖經·詩篇 107:23-24
  # Requires translation!
-for [mapUnitFilter] units = 
+for [mapUnitFilter] units = 對於[mapunitfilter]單位
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:1813]:
```suggestion
for [mapUnitFilter] units = 對於[mapUnitFilter]單位
```
Find the cause of failed check!

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 5]
```diff
@@ -3207,14 +3207,14 @@ Pileni = 皮萊尼
...
+starting from the [era] = 從[era]開始
 Enables embarkation for land units = 陸軍單位擁有船運能力
  # Requires translation!
 Enables [mapUnitFilter] units to enter ocean tiles = 
  # Requires translation!
-Normal vision when embarked = 
+Normal vision when embarked = 登船時視野正常
  # Requires translation!
-within [amount] tiles of a [tileFilter] = 
+within [amount] tiles of a [tileFilter] = 在[tilefilter]的[amount]格地塊之內
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:3217]:
```suggestion
within [amount] tiles of a [tileFilter] = 在[tileFilter]的[amount]格地塊之內
```

**Review by heipizhu4 — CHANGES_REQUESTED** [review]
Find the cause of failed check! 😄

File: android/assets/jsons/translations/Traditional_Chinese.properties (hunk 1 above)
Comment (Jia35) [reply android/assets/jsons/translations/Traditional_Chinese.properties:1327]: Thanks, i see.

File: android/assets/jsons/translations/Traditional_Chinese.properties (hunk 2 above)
Comment (Jia35) [reply android/assets/jsons/translations/Traditional_Chinese.properties:4930]: I agree.

File: android/assets/jsons/translations/Traditional_Chinese.properties (hunk 4 above)
Comment (Jia35) [reply android/assets/jsons/translations/Traditional_Chinese.properties:1813]: Thank you very much. I didn't even notice.

File: android/assets/jsons/translations/Traditional_Chinese.properties [hunk 6]
```diff
@@ -6585,15 +6585,15 @@ This means exploration and trade is important to grow your cities! = 這意味
 Nullifies [stat] [cityFilter] = 消除[cityFilter]的[stat]
 Nullifies Growth [cityFilter] = 停止[cityFilter]的人口增長
  # Requires translation!
-Provides [stats] per turn = 
+Provides [stats] per turn = 每回合提供[stats]
  # Requires translation!
-Provides [stats] [cityFilter] per turn = 
+Provides [stats] [cityFilter] per turn = 提供[stats][CityFilter]每回合
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Traditional_Chinese.properties:6590]:
```suggestion
Provides [stats] [cityFilter] per turn = 提供[stats][cityFilter]每回合
```

**Review by heipizhu4 — CHANGES_REQUESTED** [review]

**Comment by heipizhu4** [comment]
Seriously, there's even more failed check? I've got out 3 errors already and I can't find out more.
<del>Label: Help needed</del>

**Comment by heipizhu4** [comment]
Next time, remember to commit after you write about 100 lines, or it really will really drive people crazy reviewing them. :D

**Comment by Jia35** [comment]
> Next time, remember to commit after you write about 100 lines, or it really will really drive people crazy reviewing them. :D

OK 😅. I'm sorry for causing you trouble

**Comment by Jia35** [comment]
Sorry I'm not yet familiar with using GitHub.
I have found a bug, how do I edit and commits it?

**Comment by heipizhu4** [comment]
Enter your fork here and edit it. Congratlations on finding it!
image

**Review by heipizhu4 — APPROVED** [review]
Yeah!👍

**Pull request merged**